package.json

```diff
@@ -37,6 +37,7 @@
     "@babel/node": "^7.10.5",
     "@babel/plugin-transform-runtime": "^7.11.5",
     "@babel/preset-env": "^7.11.5",
+    "cors": "^2.8.5",
```

Comment:
It looks like your `package.json` is outdated. You should run `git pull origin master` onto your branch to get the latest updates.

Doing this is best practice to ensure that you're getting the latest changes before starting development.